en/notes-for-resorting.md

```diff
@@ -40,7 +40,7 @@ Buildroot is faster for initial development for new platforms as it is minimalis
 
 ### Other FAQs
 
-* https://github.com/OpenIPC/camerasrnd/blob/master/doc/FAQ-ru.md
+* [OpenIPC Wiki (ru)](../ru/index.md)
```

Comment:
Correct link here is https://github.com/OpenIPC/camerasrnd/blob/master/docs/FAQ-ru.md

Reply:
Ok, i'll fix it myself.